src/reducers/projects.js

```diff
@@ -88,6 +92,7 @@ export function reduceRoot(stateIn, action) {
 
 export default function reduceProjects(stateIn, action) {
   let state;
+  const {payload} = action;
```

Comment:
I did this to get past the line length linter 😂 But let me know if this would cause issues!

Reply:
I have a policy of not really nitpicking the reducers until after I am able to refactor them to have a reasonable structure.

Reply:
I would actually say this is a good change on the merits—in fact, I’d probably argue for just fully destructuring `action` in the params list itself!

Reply:
Ah! Right that totally makes sense! thanks!